### PR TITLE
fix: Initial typographic style changes

### PR DESCRIPTION
**Pull request by johnmcclumpha**
updates typographic styles to work with Roboto:
- Adds current (Mercury) styles 
- Removes styles to be deprecated from documentation

### CODE REVIEW

File: packages/mds/src/tailwind/base.scss [hunk 1]
```diff
@@ -62,7 +62,7 @@
 
   b,
   strong {
-    font-weight: bolder;
+    font-weight: 700;
```
Comment (mitcsutt) [inline packages/mds/src/tailwind/base.scss:65]: We'll have to make sure to store these values in style-dctionary when we add it

**Review by mitcsutt — APPROVED** [review]
lgtm

File: packages/mds/src/tailwind/base.scss (hunk 1 above)
Comment (johnmcclumpha) [reply packages/mds/src/tailwind/base.scss:65]: Yeah there's quite a bit I've picked up on along the way that we'll need to put in there - although if we just get everyone using MDS then we may not need style dictionary

**Comment by Nineswiss** [comment]
Ya proper font rendering!

**Pull request merged**